sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -661,6 +663,38 @@ export class JsonRpcProvider extends Provider {
     signature: SerializedSignature,
     requestType: ExecuteTransactionRequestType = 'WaitForEffectsCert',
   ): Promise<SuiExecuteTransactionResponse> {
+    const version = this.rpcApiVersion;
```

Comment:
Shouldn't this be `SuiExecuteTransactionResponse | SuiExecuteTransactionResponse_v26`?

Reply:
we convert `SuiExecuteTransactionResponse_v26` to `SuiExecuteTransactionResponse` in this function